packages/nice-fs/src/asyncMethods.ts

```diff
@@ -0,0 +1,123 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ *
+ * @format
+ */
+
+import fs from 'graceful-fs';
+import util from 'util';
+
+const importedMethods = [
+  'access',
+  'appendFile',
+  'chmod',
+  'chown',
+  'copyFile',
+  'lchmod',
+  'lchown',
+  'lutimes',
+  'link',
+  'lstat',
+  'mkdir',
+  'mkdtemp',
+  'open',
+  'opendir',
+  'readdir',
+  'readlink',
+  'realpath',
+  'rename',
+  'rmdir',
+  'rm',
+  'stat',
+  'symlink',
+  'truncate',
+  'unlink',
+  'utimes',
+  'writeFile',
+] as const;
+
+type ValueOf<T extends readonly any[]> = T[number];
+type ImportedMethods = ValueOf<typeof importedMethods>;
+
+export type AsyncMethods = {
+  access: typeof fs.access.__promisify__;
```

Comment:
It looks like the "__promisify__" flags have been pretty stable. https://github.com/DefinitelyTyped/DefinitelyTyped/issues/16860#issuecomment-321288482

Less worried about fragility now.